### DISCUSSION TITLE
How to specify the aggregation in the satellite accounts.

### DISCUSSION THREAD

**Discussion by ivanthung** [SESAM-Polimi/MARIO#87] [Q&A] [answered]
I have tried to use your aggregation function, but it keeps tripping up at the satellite accounts.
For example, after I have specifyied my aggregations, it tells me that: " Following item are not acceptable for level Satellite account 
 {'Domestic extraction Used - Primary Crops - Jojoba Seeds', 'Unused Domestic extraction - Primary Crops - Leguminous vegetables, nes', 'Water consumption Blue - Manufacturing - Chemicals nec', 'Unused Domestic extraction - Metal Ores - Zinc ores', 'Unused Domestic extraction - Primary Crops - Grapes', 'Unused Domestic extraction - Primary Crops - Triticale', ...... }

If they are not acceptable, how can I exclude these from the aggregations?

**Comment by nigolred** [comment, marked as answer]
Dear @ivanthung,

to exclude satellite accounts from with an aggregation you can simply pass "unused".

For example:
![image](https://github.com/it-is-me-mario/MARIO/assets/54635332/45483921-54b7-48da-9d2e-a1c1f7acdd7a)

Consider that your aggregation file must have the very same number of satellite accounts, commodity, activity, region and factor of production of your database. Therefore, use always get_aggregation_excel() function to get the Excel template.

Does this help?
